azure-pipelines.yml

```diff
@@ -0,0 +1,61 @@
+trigger:
+- V2
+
+pr:
+  branches:
+    include:
+      - V2
+
+jobs:
+- job: 'Test'
+  pool:
+    vmImage: 'ubuntu-16.04'
+  strategy:
+    matrix:
+      Python37:
+        python.version: '3.7'
+#      Python38:
+#        python.version: '3.8'
+
+  steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '$(python.version)'
+    displayName: 'Use Python $(python.version)'
+
+  - task: Bash@3
+    displayName: 'Setup pipenv'
+    inputs:
+      targetType: 'inline'
+      script: |
+        set -eux  # fail on error
+        python -m pip install --upgrade pip
+        python -m pip install pipenv
+        pipenv --python 3
+  - task: Bash@3
+    displayName: 'Install deps: Analyzer'
+    inputs:
+      targetType: 'inline'
+      workingDirectory: 'presidio-analyzer'
+      script: |
+        # Azure pipelines test publish
+        pipenv sync --dev --sequential
+        pipenv install --dev --skip-lock pytest-azurepipelines
```

Comment:
can this be taken to a different task so we can know which tasks are "core presidio" and which are "pipeline utility"?

Reply:
Fixed. moved `pipenv install --dev --skip-lock pytest-azurepipelines` to the testing task

Reply:
wouldnt we replace the "core presidio" with a build script?

Reply:
I assume this is subject to change once we have a solid build script.